src/Core/Content/Product/Cart/ProductFeatureBuilder.php

```diff
@@ -162,6 +170,9 @@ private function isRequiredCustomField(string $name, SalesChannelProductEntity $
         return false;
     }
 
+    /**
+     * @return array{label: string, value: array<mixed>, type: string}
```

Comment:
is value really an array :thinking: 

```suggestion
     * @return array{label: string, value: mixed, type: string}
```
I guess? :thinking:

Reply:
Yup, missed that :see_no_evil: